src/components/DatePickerContainer/DatePickerContainer.css

```diff
@@ -0,0 +1,11 @@
+.infoPanel {
+	padding: 10px 21px;
+	border-top: 1px solid #dce0e0;
+	color: #484848;
+}
+
+.info {
```

Comment:
Can you show me where the span is? I was looking for it, but I couldn't find it.